tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
```

Comment:
do you have a MultiQC report?

Reply:
Yes airrflow does, it is under `results/multiqc/`